content/en/docs/contribute/style/style-guide.md

```diff
@@ -210,6 +210,11 @@ The output is similar to this:
     NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
     nginx    1/1       Running   0          13s    10.200.0.4   worker0
 
+
+### Use shortcode for downloadable code blocks
```

Comment:
```suggestion
### Make code samples downloadable
```

Reply:
@anshprat , thanks for creating this PR. I am wondering if the `codenew` shortcode section belongs
in the k8s docs custom shortcode page, https://kubernetes.io/docs/contribute/style/hugo-shortcodes/?

Reply:
@sftim - thoughts on above comment from @kbhawkey ?